### DISCUSSION TITLE
Facing problem in pushing the code

### DISCUSSION THREAD

**Discussion by saha-dhriti** [community/community#143942] [Programming Help]
### Body

Hi!!! I am facing problem while trying to push code in a sub branch. "error: RPC failed; HTTP 403 curl 22 The requested URL returned error: 403" This is the error I am getting. How to resolve this issue?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mastertanzeel** [comment]
Hi @saha-dhriti,

First, check if you have the right access to the branch; if not, request access. Then, confirm that you’re using the correct credentials, especially if you recently updated your Git username or password. You can also try logging out and back in on your Git client or deleting any saved credentials. If you're using HTTPS, consider switching to SSH for authentication as an alternative.

I hope this will help you.

**Comment by dmtrinh** [comment]
This happened to me after a recent upgrade of Xcode Command Line Tools:
```
==> xcode-select --version
xcode-select version 2409.
==> git --version
git version 2.39.3 (Apple Git-145)
```

Since I was doing a large code push, I resolved the issue by bumping Git's HTTP buffer size:
```
git config --global http.postBuffer 157286400
```
